core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -55,6 +55,8 @@
     boolean hash = false;
     @Parameter(names = {"--histogram"}, description = "print a histogram of the key-value sizes")
     boolean histogram = false;
+    @Parameter(names = {"--printIndex"}, description = "prints information about all the index entries")
+    boolean printIndex = false;
```

Comment:
May also want a min depth.   I think both of the use cases can be also be achieved with a follow on grep.